src/pages/ErrorPage.jsx

```diff
@@ -1,7 +1,11 @@
 export const ErrorPage = () => {
     return (
         <div>
+          <div className="titlepage">
+
```

Comment:
prettier powinien Ci usuwać puste wiersze, ale jeśli nie usunie to nie lepiej nie dodawać ich o ile nie zwiększa to czytelności kodu